safe/gui/tools/options_dialog.py

```diff
@@ -160,6 +162,7 @@ def __init__(self, iface, parent=None, qsetting=''):
         # hide custom template dir toggle
         self.custom_templates_dir_checkbox.hide()
         self.splitter_custom_report.hide()
+        # self.set_organisation_logo()
```

Comment:
You keep it for legacy or is-it really old code?

Reply:
Just forget to delete :)